### PR TITLE
feat: creation of SurveyView

### PR DESCRIPTION
**Pull request by vhu-axelor**
RM#76316

### CODE REVIEW

File: packages/apps/intervention/src/components/pages/SurveyView/SurveyView.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          interventionId: intervention.id,
+          rangeId: selectedRangeId,
+          page: page,
+        }),
+      );
+    },
+    [dispatch, intervention.id, selectedRangeId],
+  );
+
+  const questionStatus = useCallback(
```
Comment (lme-axelor) [inline packages/apps/intervention/src/components/pages/SurveyView/SurveyView.tsx:64]: getQuestionStatus

File: packages/apps/intervention/src/components/pages/SurveyView/SurveyView.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    (state: any) => state.intervention_question,
+  );
+
+  useEffect(() => {
+    dispatch(
+      (fetchQuestion as any)({
+        interventionId: intervention.id,
+        rangeId: selectedRangeId,
+      }),
+    );
```
Comment (lme-axelor) [inline packages/apps/intervention/src/components/pages/SurveyView/SurveyView.tsx:48]: Maybe we can use fetchQuestionAPI here

File: packages/apps/intervention/src/components/atoms/QuestionCard/QuestionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    [Colors, status],
+  );
+
+  return (
+    <ObjectCard
+      style={styles.border}
+      onPress={onPress}
+      upperTexts={{
+        items: [
+          {displayText: interventionRange?.name, hideIfNull: true},
```
Comment (lme-axelor) [inline packages/apps/intervention/src/components/atoms/QuestionCard/QuestionCard.tsx:52]: use interventionRange.rangeVal.title here

File: packages/apps/intervention/src/screens/intervention/InterventionDetailsScreen.tsx [hunk 4]
```diff
@@ -41,6 +41,11 @@ const InterventionDetailsScreen = ({route}) => {
       viewComponent: <GeneralInformationView />,
       color: Colors.secondaryColor_dark,
     },
+    {
+      iconName: 'card-checklist',
+      viewComponent: <SurveyView />,
+      color: Colors.progressColor,
```
Comment (lme-axelor) [inline packages/apps/intervention/src/screens/intervention/InterventionDetailsScreen.tsx:47]: Should be disabled if status is under started

**Review by gca-axelor — APPROVED** [review]

**Pull request merged**